kolibri/core/assets/src/vue/core-base.vue

```diff
@@ -93,23 +95,13 @@
         return this.mobile ? 56 : 64;
       },
       navWidth() {
-        return 270;
-      },
-      navPadding() {
-        const PADDING = 32;
-        if (this.mobile || !this.navShown) {
-          return PADDING;
-        }
-        return this.navWidth + PADDING;
+        return this.navShown ? this.headerHeight * 4 : 0;
       },
-      navStyle() {
-        if (this.navShown) {
-          return { paddingLeft: `${this.navPadding}px` };
-        }
-        return '';
+      appBarStyle() {
+        return { paddingLeft: `${this.navWidth + PADDING}px` };
       },
       contentStyle() {
-        return { left: `${this.navPadding}px`, top: `${this.headerHeight}px` };
+        return { left: `${this.navWidth}px`, top: `${this.headerHeight}px` };
```

Comment:
When closed, the app-bar should have it's default `16px` left-padding, not `32px`.